### PR TITLE
Add Lepton as featured app on Electron website

### PR DESCRIPTION
**Pull request by hackjutsu**
Here is our app **Lepton**.

It is a GitHub Gist client built on Electron. We provide free macOS/Windows/Linux downloads on our [release](https://github.com/hackjutsu/Lepton/releases) page.

Please let me know if there is something missing in this PR.

Thanks,

-Cosmo

### CODE REVIEW

File: apps/lepton/lepton.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Lepton
+description: 'Open-source GitHub Gist client built for macOS, Windows and Linux. '
```
Comment (zeke) [inline apps/lepton/lepton.yml]: There's an extra space at the end of the description. Could probably live without the "for macOs... " part too.

Comment (hackjutsu) [inline apps/lepton/lepton.yml]: Thanks for feedback. I'll remove the extra space and the "for macOS part".

**Comment by hackjutsu** [comment]
@zeke Haha, that's interesting~ 

That's my English name I picked up around 2008 before I moved to US. 
Are you living in SB? Me too!

**Comment by zeke** [comment]
> Are you living in SB?

I was! But I moved yesterday. Moved down to Moorpark / Thousand Oaks with my family.

Next time I'm in SB we should have coffee. ☕️

**Comment by hackjutsu** [comment]
@zeke Sure thing!

**Pull request merged**